docs/hello_nextflow/02_hello_world.md

```diff
@@ -838,7 +849,27 @@ _After:_
 greeting_ch = Channel.fromPath(params.input_file).splitText() { it.trim() }
 ```
 
-#### 2. Run the workflow with the `-ansi-log false` option and an `--input_file` parameter
+#### 2. Modify the default parameter to point to an input file
+
+_Before:_
+
+```groovy title="hello-world.nf" linenums="38"
+/*
+ * Pipeline parameters
+ */
+params.greeting = "Bonjour le monde!"
+```
+
+_After:_
+
+```groovy title="hello-world.nf" linenums="38"
+/*
+ * Pipeline parameters
+ */
+params.input_file = "data/greetings.txt"
+```
+
+#### 3. Run the workflow with the `-ansi-log false` option and an `--input_file` parameter
 
 ```bash
 nextflow run hello-world.nf -ansi-log false --input_file data/greetings.txt
```

Comment:
_(Sticking this comment here because github won't let me add a comment further in the unchanged bit)_

This tip:
`!!! tip
    But don't worry if this feels like a lot to grapple with all of a sudden! This is just meant to be a little peek at the kind of things you will learn in later training modules.`
feels well-intentioned but a bit cryptic. I would rephrase it to say more specifically "Don't worry if the channel types and operators, closures etc feel like a lot to grapple with the first time you encounter them. You'll get more opportunities to practice using these components in various settings in later training modules."